component/module/string/string.go

```diff
@@ -6,13 +6,15 @@ import (
 	"github.com/grafana/agent/component"
 	"github.com/grafana/agent/component/module"
 	"github.com/grafana/agent/pkg/river/rivertypes"
+	"github.com/grafana/agent/service/http"
 )
 
 func init() {
 	component.Register(component.Registration{
-		Name:    "module.string",
-		Args:    Arguments{},
-		Exports: module.Exports{},
+		Name:          "module.string",
+		Args:          Arguments{},
+		Exports:       module.Exports{},
+		NeedsServices: []string{http.ServiceName},
```

Comment:
Why does this module loader need the HTTP service? Is it just to expose information for the UI?